sources/camunda-docker-compose-core.yml

```diff
@@ -178,3 +178,4 @@ volumes:
 
 networks:
   local-network:
+    driver: bridge
```

Comment:
Was it missing or just omitted because it is the default?
It this related to Redis or can you clarify what this is fixing or changing?

Reply:
no not redis related, i can remove it.